synapse/cores/common.py

```diff
@@ -769,14 +779,22 @@ def _actNodeDel(self, mesg):
 
         self.delTufo(node)
 
-    def _actNodeSet(self, mesg):
+    def _actNodePropSet(self, mesg):
         form = mesg[1].get('form')
         valu = mesg[1].get('valu')
         props = mesg[1].get('props')
 
         node = self.formTufoByProp(form, valu)
         self.setTufoProps(node, **props)
 
+    def _actNodePropDel(self, mesg):
+        form = mesg[1].get('form')
+        valu = mesg[1].get('valu')
+        prop = mesg[1].get('prop')
```

Comment:
If we can set multiple props at once, should we be able to delete multiple props at once?